e2e/tests/workspace/workspace-edit.spec.ts

```diff
@@ -11,6 +11,10 @@ describe('Editing workspace via workspace card snowman menu', () => {
     await signIn(page);
   });
 
+  // Reuse same Workspace for all tests in this file to reduce test playback time.
+  // Workspace to be created in first test. If first test fails, next test will create it.
+  let workspaceName: string;
```

Comment:
should we just create/find a workspace in a `beforeAll` function? I'm not sure if its always the case that tests will run in sequence, especially if we add parallelization.

Reply:
In Jest, tests in same file will always run. in serial order.